grab.py

```diff
@@ -9,15 +10,45 @@
 import PIL.ImageDraw
 
 
-def capture_frame():
+def get_images(base_filename):
     """
-    Capture an image from the camera. Return a numpy ndarray with the image
-    data.
+    Step through cameras, capturing images.
     """
-    cam = cv2.VideoCapture(0)
+    camCounter = 0
```

Comment:
Just a small detail: In the almost universally used [PEP8](https://www.python.org/dev/peps/pep-0008/) style guide `under_scores` are recommended for variables instead of `camelCase`.